<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTestClass</class>
 <widget class="QMainWindow" name="QTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>242</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tic Tac Toe</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Play!</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QSpinBox" name="gameSize">
        <property name="minimum">
         <number>3</number>
        </property>
        <property name="maximum">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Mode:</string>
        </property>
        <widget class="QRadioButton" name="firstPlayer">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>222</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Player first</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="firstCPU">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>222</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>CPU first</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="pvp">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>222</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Player vs Player</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>10</height>
         </size>
        </property>
        <property name="text">
         <string>Game Size:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qtest.qrc"/>
 </resources>
 <connections/>
</ui>
